client/state/media/actions.js

```diff
@@ -192,11 +194,13 @@ export const editMediaItem = ( siteId, mediaItem, data ) => ( {
  *
  * @param {number} siteId site identifier
  * @param {object} item media item
+ * @param {object} originalMediaItem media item without updated properties
  */
-export const updateMediaItem = ( siteId, item ) => ( {
+export const updateMediaItem = ( siteId, item, originalMediaItem ) => ( {
```

Comment:
Is there a chance that the ID of the `originalMediaItem` and `item` will be different? Seems like we only use the `ID` from `originalMediaItem`.

Also, if we only use the ID, should we pass only the ID as an argument here?

Reply:
See my comment here https://github.com/Automattic/wp-calypso/pull/46778#discussion_r512690042